test/langPickerTest.js

```diff
@@ -4,31 +4,167 @@ const assert = require('assert');
 const LanguagePicker = require('../lib/LanguagePicker');
 
 describe('LanguagePicker', () => {
-  function test(languages, expected, ...testVals) {
-    return () => {
-      const lp = new LanguagePicker({
+  const cases = [
+    {
+      msg: 'No values given',
+      langCode: 'en',
+      values: undefined,
+      expected: undefined,
+    },
+    {
+      msg: 'Pick first (only) value',
+      langCode: 'en',
+      values: [
+        { en: '_en' },
```

Comment:
just to reduce possible confusion - don't use "_en" in the values because "_" is used in the code, and "en" is the same as the key. Do something like  "*EN" - will make things stand out.